src/main/groovy/com/lesfurets/jenkins/unit/BasePipelineTest.groovy

```diff
@@ -211,6 +211,10 @@ abstract class BasePipelineTest {
         helper.registerAllowedMethod("stage", [String, Closure])
         helper.registerAllowedMethod("step", [Map])
         helper.registerAllowedMethod("string", [Map], stringInterceptor)
+        helper.registerAllowedMethod('timeout', [Integer, Closure], { Map args, Closure c ->
```

Comment:
Hello @nre-ableton !
`[Integer, Closure], { Map args, Closure c ->` is not type safe, better to use `[Integer, Closure], { Integer i, Closure c ->` here.